tests/test_multiline.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+from mssqlcli.mssqlbuffer import _is_complete
+
+
+class TestMssqlCliMultiline:
+    testdata = [
+        (None, False),
+        ('', False),
+        ('select 1 /* open comment!\ngo', False),
+        ('select 1\ngo -- another comment', True),
+        ('select 1; select 2, "open quote: go', False),
+        ('select 1\n"go"', False),
+        ('select 1; GO', False),
+        ('SELECT 4;\nGO', True),
+        ('select 1\n select 2;\ngo', True),
+        ('select 1;', False),
+        ('select 1 go', False),
+        ('select 1\ngo go go', False)
+        # tests below to be enabled when sqlparse supports retaining newlines
+        # when stripping comments (tracking here: https://github.com/andialbrecht/sqlparse/issues/484):
+        # ('select 3 /* another open comment\n*/   GO', True),
+        # ('select 1\n*/go', False),
+        # ('select 1 /*\nmultiple lines!\n*/go', True)
+    ]
+
+    @staticmethod
```

Comment:
please put some comments here around what this test is doing exactly. someone may look at this 5 years from now and be a little confused what is_complete refers to 😄